website/src/common/DailySummary.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2024 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import {
+  ChartConfig,
+  ChartContainer,
+  ChartTooltip,
+  ChartTooltipContent,
+} from "@/components/ui/chart";
+import { DailySummarydata } from "@/types";
+import PropTypes from "prop-types";
+import { Line, LineChart, XAxis } from "recharts";
+
+export type DailySummaryProps = {
+  data: DailySummarydata;
+  benchmarkType: string;
+  setBenchmarktype: (type: string) => void;
+};
+
+export default function DailySummary({ data }: DailySummaryProps) {
+  const chartData: any[] | undefined = [];
+
+  const chartConfig = {
+    desktop: {
+      label: "Desktop",
+      color: "hsl(var(--primary))",
+    },
+    mobile: {
+      label: "Mobile",
+      color: "hsl(var(--primary))",
+    },
+  } satisfies ChartConfig;
+
+  if (data.data !== null) {
+    data.data.map((item) => ({
+      totalQps: chartData.push({
+        totalQps: item.total_qps.center / 2,
```

Comment:
I am not sure I follow why we are dividing the value by two here. It will yield wrong results.

Reply:
I was doing some test to test the height of the chart, I forgot to remove this, thank you!